pages/linux/logwatch.md

```diff
@@ -0,0 +1,11 @@
+# logwatch
+
+> Analyzes and summarizes system logs.
```

Comment:
What does "system" mean here ? syslog ? Or any application log ? If its the latter, where is the location specified ?